### PR TITLE
refactor(child-writer): add robot commands constant

### PR DESCRIPTION
**Pull request by jviotti**
See: https://github.com/resin-io/etcher/pull/1295#discussion_r112463686
Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

File: lib/child-writer/index.js [hunk 1]
```diff
@@ -127,6 +127,7 @@ exports.write = (image, drive, options) => {
    */
   const bridgeRobotMessage = (message) => {
     const ROBOT_COMMAND_LOG = 'log';
+    const ROBOT_COMMAND_ERROR = 'error';
```
Comment (jhermsmeier) [inline lib/child-writer/index.js]: Agree; `robot.COMMAND_LOG` would follow the usual pattern of how constants come from places

Comment (jviotti) [inline lib/child-writer/index.js]: Yeah, lets do it!

Comment (jviotti) [inline lib/child-writer/index.js]: Done!

File: lib/child-writer/index.js [hunk 2]
```diff
@@ -136,7 +134,7 @@ exports.write = (image, drive, options) => {
       // Don't be so strict. If a message doesn't look like
       // a robot message, then make the child writer log it
       // for debugging purposes.
-      return robot.parseMessage(robot.buildMessage(ROBOT_COMMAND_LOG, {
+      return robot.parseMessage(robot.buildMessage(robot.COMMANDS.LOG, {
```
Comment (jhermsmeier) [inline lib/child-writer/index.js]: To stick with the conventional naming of enums, should this be singular, rather than plural, i.e. `robot.COMMAND.LOG`?

Comment (jviotti) [inline lib/child-writer/index.js]: Fixed!

File: lib/shared/robot/index.js [hunk 3]
```diff
@@ -255,5 +282,5 @@ exports.printMessage = (message, data) => {
  * robot.log({ example: 'data' });
  */
 exports.log = (data) => {
-  exports.printMessage('log', data);
+  exports.printMessage(exports.COMMAND.LOG, data);
```
Comment (lurch) [inline lib/shared/robot/index.js]: :tada:

File: tests/shared/robot.spec.js [hunk 4]
```diff
@@ -22,6 +22,23 @@ const robot = require('../../lib/shared/robot');
...
+      m.chai.expect(_.every(_.values(robot.COMMAND), _.isString)).to.be.true;
+    });
+
+    it('should contain only unique values', function() {
+      const numberOfKeys = _.size(_.keys(robot.COMMAND));
+      m.chai.expect(_.size(_.uniq(_.values(robot.COMMAND)))).to.equal(numberOfKeys);
+    });
+
+  });
+
```
Comment (lurch) [inline tests/shared/robot.spec.js]: Given that later tests rely on `command: 'error'`, perhaps it'd be worth checking here that `robot.COMMAND.ERROR === 'error'` ?

(and it also looks like there are no tests for `robot.log`?)

Comment (jviotti) [inline tests/shared/robot.spec.js]: Fixed! `robot.log` only prints to `stdout`, which is quite tricky to test. It internally uses `.buildMessage`, which is already well tested.

**Pull request merged**